docs/en/ingest-management/fleet/fleet-server-monitoring.asciidoc

```diff
@@ -53,5 +52,10 @@ metric and then resize the {fleet-server}, if necessary.
 [role="screenshot"]
 image::images/dashboard-datastream.png[Dashboard Data stream]
 
+//REVIEWERS: This dashboard is missing CPU usage info. Is that expected? Also
```

Comment:
I have tested this locally and I also didn't get any CPU or memory metrics, I have raised the question with the integrations team, definitely looks like something isn't working correctly.

Reply:
OK. I think I will leave the docs as-is since it seems that this is a product bug.